docs/sources/visualizations/canvas.md

```diff
@@ -0,0 +1,65 @@
+---
+aliases:
+  - /docs/grafana/latest/features/panels/canvas/
+  - /docs/grafana/latest/visualizations/canvas/
+description: Canvas visualization documentation
+keywords:
+  - grafana
+  - canvas
+  - panel
+  - documentation
+title: Canvas
+weight: 600
+---
+
+# Canvas
+
+Canvas is a new panel that combines the power of Grafana with the flexibility of custom elements. Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren't possible with standard Grafana panels, all within Grafana's UI. If you've used popular UI and web design tools, then designing Canvas panels will feel very familiar.
+
+{{< figure src="/static/img/docs/canvas-panel/canvas-beta-overview-9-2-0.mov" max-width="750px" caption="Canvas panel beta overview" >}}
```

Comment:
I think something funky with the .mov files. -- should they be something else?

When I try loading locally I see: 
<img width="609" alt="image" src="https://user-images.githubusercontent.com/705951/194998144-556853ce-0eac-4153-8caa-36492e5b58fb.png">

but other images elsewhere work 🤔

Reply:
replaced the `figure` tag with `video-embed` - it does work now